bot.js

```diff
@@ -16,7 +16,7 @@ const commandPrefix = '~';
 const botToken = '..';
```

Comment:
This will change later and will be put on the env file but why is there a .. inside ' '